### ISSUE TITLE
Media source can't loop .aac files.

### ISSUE THREAD

**Issue by CodeYan01** [obsproject/obs-studio#6715]
### Operating System Info

Windows 10

### OBS Studio Version

27.2.4

### OBS Studio Log URL

https://obsproject.com/logs/vI_MqxXZfSqKO25F

### Expected Behavior

Media sources with .aac files and loop enabled should loop.

### Current Behavior

When it reaches the end, the timer simply goes back to 00:00:00 / -00:00:31, but the source still isn't stopped, as in dock shows a pause button and a stop sign. Even if you stop it and restart, or switch to different scene and go back, the source will not properly play (timer still doesn't move, but pause button shows). To actually fix the source, you have to seek to some random time, which won't work as obs puts it back to 00:00:00. You can then stop and restart the source. Note that the log says `MP: Failed to seek: Operation not permitted`, but seeking works normally when the source has not reached the end yet.

### Steps to Reproduce

1. New scene collection
2. Have a video file. I just put a video in obs and recorded it (just to make it so the file came from obs, to reduce variables here.
3. Remux to m4a via `ffmpeg -i INPUT_FILE -c copy OUTPUT_FILE.aac`.
4. Step 3 again but remux to .m4a. (also tried .mp4 file output with only an audio track, skipped for brevity here, but works as fine as the .m4a)
5. Add 3 scenes. 
6. In the first one, drop the original video.
7. In the second one, drop the .aac file
8. In the third one, drop the .m4a. 
9. Enable the Loop option for all three sources.
10. Test them one by one. Make it play to the end and verify that the loop works. Looping will not work for the .aac file.
...

### Anything else we should know?

I was wondering if the file extension has something to do with this bug, so I tried to rename the .aac file to .m4a (simple rename, not remux). The symptom still occurred, so I think it has something to do with the .aac file container rather than the literal file extension.

I tried to loop the .aac file in VLC and PotPlayer, and they can loop the file fine.

Log also says `warning: Estimating duration from bitrate, this may be inaccurate` upon adding the media source for the .aac file.

**Comment by tuduweb** [comment]
I reproduce this BUG in ubuntu20, this is a BUG caused by `fmt->start_time`.

When the format is AAC, the start_time is `N/A` in ffmpeg.

```
 bin@bin-iMacPC  ~  ffprobe -i newboy.aac -show_streams
ffprobe version 5.0 Copyright (c) 2007-2022 the FFmpeg developers
  built with Apple clang version 13.0.0 (clang-1300.0.29.3)
  configuration: --prefix=/usr/local/Cellar/ffmpeg/5.0 --enable-shared --enable-pthreads --enable-version3 --cc=clang --host-cflags= --host-ldflags= --enable-ffplay --enable-gnutls --enable-gpl --enable-libaom --enable-libbluray --enable-libdav1d --enable-libmp3lame --enable-libopus --enable-librav1e --enable-librist --enable-librubberband --enable-libsnappy --enable-libsrt --enable-libtesseract --enable-libtheora --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libxvid --enable-lzma --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libspeex --enable-libsoxr --enable-libzmq --enable-libzimg --disable-libjack --disable-indev=jack --enable-videotoolbox
  libavutil      57. 17.100 / 57. 17.100
  libavcodec     59. 18.100 / 59. 18.100
  libavformat    59. 16.100 / 59. 16.100
  libavdevice    59.  4.100 / 59.  4.100
  libavfilter     8. 24.100 /  8. 24.100
  libswscale      6.  4.100 /  6.  4.100
  libswresample   4.  3.100 /  4.  3.100
  libpostproc    56.  3.100 / 56.  3.100
[aac @ 0x7f8a2dd09f40] Estimating duration from bitrate, this may be inaccurate
Input #0, aac, from 'newboy.aac':
  Duration: 00:04:23.21, bitrate: 127 kb/s
  Stream #0:0: Audio: aac (LC), 44100 Hz, stereo, fltp, 127 kb/s
[STREAM]
index=0
codec_name=aac
codec_long_name=AAC (Advanced Audio Coding)
profile=LC
codec_type=audio
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
sample_fmt=fltp
sample_rate=44100
channels=2
channel_layout=stereo
bits_per_sample=0
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/28224000
start_pts=N/A
start_time=N/A
duration_ts=7428937733
duration=263.213497
bit_rate=127023
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
DISPOSITION:captions=0
DISPOSITION:descriptions=0
DISPOSITION:metadata=0
DISPOSITION:dependent=0
DISPOSITION:still_image=0
[/STREAM]
```

**Comment by gxalpha** [comment]
This should be fixed by #6735
